contracts/modules/common/Utils.sol

```diff
@@ -35,13 +35,13 @@ library Utils {
         // we jump 32 (0x20) as the first slot of bytes contains the length
         // we jump 65 (0x41) per signature
         // for v we load 32 bytes ending with v (the first 31 come from s) then apply a mask
-        // solium-disable-next-line security/no-inline-assembly
+
         assembly {
             r := mload(add(_signatures, add(0x20,mul(0x41,_index))))
             s := mload(add(_signatures, add(0x40,mul(0x41,_index))))
             v := and(mload(add(_signatures, add(0x41,mul(0x41,_index)))), 0xff)
         }
-        require(v == 27 || v == 28); // solium-disable-line error-reason
+        require(v == 27 || v == 28);
```

Comment:
I would add `"reason-string": ["warn", {"maxLength":32}]` to the ruleset and add `solhint-disable-next-line reason-string` for that particular line as a reminder to revisit whether we have a good reason not to include a reason here :)

Reply:
I'd even make it an error